controller/internal/routes/authenticate_router.go

```diff
@@ -102,3 +105,18 @@ func (ro *AuthRouter) authHandler(ae *env.AppEnv, rc *response.RequestContext) {
 
 	rc.RequestResponder.RespondWithOk(currentSession, nil)
 }
+
+func (ro *AuthRouter) mapConfigTypeNamesToIds(ae *env.AppEnv, values []string) []string {
+	var result []string
+	if stringz.Contains(values, "all") {
```

Comment:
let's say we internationalize this one day - would 'all' still be the special token? not sure if that's something to consider now or not but internationalization has been rolling around in my head for a while so i'm making the comment

Reply:
Probably handle that with some sort of layer of indirection, so that we use some special token under the hood, but allow different types to be mapped to it.

The prefixing could also be tidied at some point, so that if we want to change the syntax we don't have to do as much hunting.